zebedee.nomad

```diff
@@ -49,6 +49,9 @@ job "zebedee" {
           "-Xmx{{PUBLISHING_RESOURCE_HEAP_MEM}}m",
           "-cp target/dependency/*:target/classes/",
           "-Drestolino.classes=target/classes",
+          "-javaagent:target/dependency/aws-opentelemetry-agent-1.30.0.jar",
```

Comment:
Should this be aws-opentelemetry-agent-1.31.0.jar for consistency with the babbage version?